pkgs/by-name/ut/ut1999/package.nix

```diff
@@ -94,6 +99,11 @@ in stdenv.mkDerivation rec {
       runHook postInstall
     '';
 
+  # .so files in the SystemARM64 directory are not loaded properly on aarch64-linux
+  appendRunpaths = lib.optional (stdenv.hostPlatform.system == "aarch64-linux") [
```

Comment:
I can imagine this works, but I think it'd be a bit more correct to use <code>lib.optional<b>s</b></code>. It doesn't matter now because it is only one item in the array, but perhaps in the future.